CI/scripts/replace_algname.sh

```diff
@@ -0,0 +1,191 @@
+#!/bin/bash
+# #############################################################################
+# Simple script to take a list of input tokens and replace the algorithm name
+# with a specified new-name. This script receives a mapping data-file which
+# specifies the prefix of the new name.
+#
+# Example:
+#   "aes-128"   -> enc_method_aes_128
+#   "sha256"    -> digest_sha256
+#
+# This replacement rule will be specified by a line as follows in the map file:
+#
+# "aes-128" : encode_method
```

Comment:
These comments need to be updated. Final layout of the mapping entry changed ...